### PR TITLE
[Refactor] Theme attributes name change

### PR DESCRIPTION
**Pull request by Mrayyan**
## Purpose

* Theme attribute refactoring to match with iOS
* Renamed file to theme (for attribute file)
* Made attr public (applicable styles/themes are in public.xml)
* Only one theme (merged 2 base themes into 1)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Open...
* Click on...

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by Mrayyan** [comment]
Removing update to [AzureCommunicationUI.Theme.Calling.md](https://github.com/Azure/communication-ui-library-android/pull/208#diff-f22b0fed63e9065f232e37dda155890b143bae3a653cf964d680d75f099ce6fc) since this change is still not available to the public, only in dev/beta for now. Will send PR for Quickstart and documentation update later.

**Review by ShaunaSong — DISMISSED** [review]

**Review by Mohtasim — DISMISSED** [review]

**Review by ahammer — DISMISSED** [review]

**Review by ahammer — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 ### New Features
```
Comment (iaulakh) [inline CHANGELOG.md:5]: please also include theming tint color options

File: docs/api/CallComposite/resources/AzureCommunicationUI.Theme.Calling.md [hunk 2]
```diff
@@ -8,6 +8,6 @@ Theme configuration to provide theme id from application to CallComposite.
 
 ## Colors
```
Comment (iaulakh) [inline docs/api/CallComposite/resources/AzureCommunicationUI.Theme.Calling.md:9]: We can also include tint colors here.

**Review by iaulakh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Mrayyan) [reply CHANGELOG.md:5]: @iaulakh It was added before and this pr is only a name change. Just to confirm, should I add it?

Comment (iaulakh) [reply CHANGELOG.md:5]: Yes @Mrayyan , it will help us to remind theming feature during release.

**Review by lpeixin — DISMISSED** [review]

**Review by phileo99 — DISMISSED** [review]
👍

**Review by ahammer — APPROVED** [review]

**Review by iaulakh — APPROVED** [review]

**Pull request merged**